cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/DependencyFormatValidator.java

```diff
@@ -0,0 +1,20 @@
+package io.cloudslang.lang.compiler.modeller.transformers;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Created by Genadi Rabinovich, genadi@hpe.com on 23/06/2016.
+ */
+class DependencyFormatValidator {
+    private static final int DEPENDENCY_PARTS = 3;
+    public static final String INVALID_DEPENDENCY = "Dependency definition should contain exactly [" + DEPENDENCY_PARTS + "] non empty parts separated by ':'";
+    void validateDependency(String dependency) {

```

Comment:
maybe add this method to the `PreCompileValidator` so all the validations will be in one central place - WDYT?